graphics/graphviz-devel/Portfile

```diff
@@ -2,8 +2,9 @@
 
 PortSystem                      1.0
 PortGroup                       xcodeversion 1.0
+PortGroup                       deprecated 1.0
```

Comment:
I'm not sure I agree with deprecating the Devel port. It can be helpful when testing upstream changes, before rolling them out to the masses. And we maintain others as well, including for ports like Cairo.

Reply:
IMO, the way the devel port was implemented to coincide with the already complicated graphviz port requires too much overhead to be worth the advantages. Consider that it has been years since the graphviz port has been updated and months since there has been any action on this PR. If someone wants to maintain the devel port, great, but please don't let it hold up updates to the base graphviz port.